Example/Tests/OHAlphabeticalSortPostProcessorTests.m

```diff
@@ -60,6 +66,26 @@ - (void)setUp
     self.contactC.lastName = @"B";
 
     self.contactD = [[OHContact alloc] init];
```

Comment:
Could you make the pattern for test case (variable names -> variable values) consistent?
Could you add a test case set with non-single-character names?